nextflow.config

```diff
@@ -252,7 +252,7 @@ manifest {
     description     = 'Amplicon sequencing analysis workflow using DADA2 and QIIME2'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version         = '2.4.0'
+    version = '2.5.0dev'
```

Comment:
Should the `=` be inline with the ones above?

Reply:
Absolutely, thanks, will correct it.

Reply:
done